pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/QualifiedName.java

```diff
@@ -5,59 +5,72 @@
 package net.sourceforge.pmd.lang.java.ast;
 
 import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
- * Represents Qualified Names for use within PackageStats
- * TODO make unit tests once the visitor is working to ensure new implementations won't break it
+ * Represents Qualified Names for use within PackageStats.
+ * TODO:cf make unit tests once the visitor is working to ensure new implementations won't break it
```

Comment:
A qualified name uniquely identifies a class, a nested class, a constructor or a method. It's basically the id, with which you can lookup metrics later on for a specific class/method. Did I get this right?